packages/libs/react-ui/src/components/Form/TextField/TextField.css.ts

```diff
@@ -0,0 +1,70 @@
+import { style } from '@vanilla-extract/css';
+import { atoms, bodyBaseRegular, token } from '../../../styles';
+
+export const startAddon = style({
+  position: 'absolute',
+  insetBlockStart: '50%',
+  transform: 'translateY(-50%)',
+  insetInlineStart: token('spacing.sm'),
+});
+
+export const endAddon = style({
+  position: 'absolute',
+  insetBlockStart: '50%',
+  transform: 'translateY(-50%)',
+  insetInlineEnd: token('spacing.sm'),
+});
+
+export const input = style([
+  bodyBaseRegular,
+  atoms({
+    borderRadius: 'sm',
+    backgroundColor: 'layer-3.default',
+    border: 'none',
+    color: 'text.base.default',
+    outline: 'none',
+    flex: 1,
+    fontSize: 'base',
+    overflow: 'hidden',
+    lineHeight: 'lg',
+  }),
+  {
+    paddingInlineStart: token('spacing.md'),
+    paddingInlineEnd: token('spacing.md'),
+    paddingBlock: token('spacing.sm'),
```

Comment:
I feel like a lot of these styles are duplicates. If so, can we pull them into a shared class so that any changes will automatically be applied to all of the form components?

Reply:
I see the same but I kept them separate to avoid unintentionally breaking other stuff, maybe later we can dry those a bit